lang/LangPrimSource/PyrListPrim.cpp

```diff
@@ -471,6 +471,9 @@ int prSymbol_envirPut(struct VMGlobals* g, int numArgsPushed) {
 
     if (!ISKINDOF(dict, class_identdict_index, class_identdict_maxsubclassindex))
         return errFailed;
+    
```

Comment:
It seems that there's an extra space here that triggers the linting failure

Reply:
extra whitespaces before the line break … (thanks!)